### ISSUE TITLE
[Regression] EditorConfig templates are missing their display name

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#38209]
**Version Used**: 
Version 16.3.0 Preview 3.0 [29221.207.master]

**Steps to Reproduce**:

**Expected Behavior**:
![image](https://user-images.githubusercontent.com/1103906/63518698-f0c62700-c534-11e9-8faa-5b39c265306d.png)

**Actual Behavior**:

16.2:
![image](https://user-images.githubusercontent.com/1103906/63518757-10f5e600-c535-11e9-88bf-89cf5a627927.png)

**Comment by davkean** [comment]
Same for VB:

![image](https://user-images.githubusercontent.com/1103906/63520037-5adfcb80-c537-11e9-9b7e-0b67b1fad8a7.png)

tag @tmeschter @jasonmalinowski

**Comment by jasonmalinowski** [comment]
Yeah I've been seeing this too. @mikadumot or @olegtk any idea where this template lives?

**Comment by tmeschter** [comment]
I think those live in https://github.com/dotnet/templates.
